doc/build/changelog/unreleased_20/11305.rst

```diff
@@ -0,0 +1,6 @@
+.. change::
+    :tags: bug, typing
```

Comment:
**Michael Bayer** (zzzeek) wrote:

this is an ORM runtime issue, not typing

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5260

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5260